js/services/ObjectData.ts

```diff
@@ -21,7 +21,7 @@ const ObjectData = {
             };
         }
 
-        let filteredObjectData = clone(objectData);
+        let filteredObjectData = clone(objectData.map((obj) => { return obj.objectData; }));
```

Comment:
This is a little confusing as it isn't actually filtered at this point, maybe give the unfiltered and filtered lists separate variables?
Also, is this mapping objectData into a list of objectData from within each object inside objectData?
Is objectData nested? Is objectData sometimes an array and sometimes an object?

Reply:
Done